### PR TITLE
[New Exercise]: Simple Cipher

### PR DESCRIPTION
**Pull request by kahgoh**
Looking to add the Simple Cipher exercise for [Object Oriented October](http://forum.exercism.org/t/select-featured-exercises-for-object-oriented-october/7112/18).

### CODE REVIEW

**Comment by meatball133** [comment]
To simplify the exercise creations, we generate from a template file, it often also makes the creation of the test file faster compared to writing it manually. And we can make changes with very little effort. Here is a doc on the Crystal test generator: https://github.com/exercism/crystal/blob/main/docs/generator.md. When writing the template, I recommend looking at how the other templates have been written and we do use ecr, which is the built-in template engine in Crystal. So there are no dependencies for running the script.

**Comment by meatball133** [comment]
I could write you a template for this exericse if you like, but I would recommend to try it yourself.

**Comment by kahgoh** [comment]
I'll have a go at creating the template ;)

File: exercises/practice/simple-cipher/src/simple_cipher.cr [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class SimpleCipher
+  def key
+    raise "Please implement the key method"
```
Comment (meatball133) [inline exercises/practice/simple-cipher/src/simple_cipher.cr:3]: Just one tiny nit, I think writing the stub file like this is nothing wrong with, and is the style for the concept exercises. However, the practice exercises don't use this style but only have a comment and don't use raise. like here: https://github.com/exercism/crystal/blob/main/exercises/practice/atbash-cipher/src/atbash_cipher.cr.

So for consistency, I would like that style to be used:
```
class SimpleCipher
   # Write your code for the 'Simple Cipher' exercise in this file.
  def key
  end

  def encode(text : String)
  end

  def decode(text : String)
  end
end
```

**Review by meatball133 — COMMENTED** [review]
Nice job with the template, looking great, just one tiny nit

**Review by meatball133 — APPROVED** [review]

**Pull request merged**